### DISCUSSION TITLE
RTL support

### DISCUSSION THREAD

**Discussion by ugurc4n** [tailwindlabs/tailwindcss#10954] [Help]
In version 3.3, if I want to give different values for ltr and rtl, which prop should I use? ltr and rtl do not work in 3.3

**Comment by RobinMalfait** [comment]
You can use the the `ltr` and `rtl` variants, this requires a `dir="rtl"` or `dir="ltr"` (default) in the template to trigger the change.

https://tailwindcss.com/docs/hover-focus-and-other-states#rtl-support

We also introduced logical properties that should help with this: https://tailwindcss.com/blog/tailwindcss-v3-3#simplified-rtl-support-with-logical-properties

If this doesn't answer your question, could you then elaborate what you want to achieve? A minimal reproduction repo or https://play.tailwindcss.com could help to figure out what's going on.

Hope this helps!

**Reply by dohahelmy** [reply]
It doesn't work.
In projects, classes works on development but doesn't on production

**Comment by ugurc4n** [comment]
rtl and ltr are not readable after build. ltr and rtl do not work when I try through https://play.tailwindcss.com/

**Reply by ugurc4n** [reply]
Is there any progress on this issue? @RobinMalfait

**Reply by RobinMalfait** [reply]
Can you share a minimal reproduction about what you mean exactly?

**Reply by ugurc4n** [reply]
Let's say we want to use two different values for ltr and rtl. for example ltr:mr-2 rtl:ml-3 in this case the ltr and rtl values I write are not read during build

**Reply by ugurc4n** [reply]
<img width="622" alt="image" src="https://user-images.githubusercontent.com/32024083/234213721-06da781c-340a-41dc-aa33-9af533bfe971.png">
<img width="327" alt="image" src="https://user-images.githubusercontent.com/32024083/234213785-4e323412-88ee-491d-bda7-c2911dbe5d8d.png">

Hi again, is there any progress on this problem, I am attaching the pictures as a more illustrative example, it does not work when I want to use different values for ltr and rtl and when I write it this way. @RobinMalfait
